molecule/default/tests/test_create_and_assign_floating_ip.py

```diff
@@ -0,0 +1,86 @@
+import pytest_rpc as helpers
+import os
+import pytest
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('os-infra_hosts')[:1]
+
+utility_container = ("lxc-attach -n $(lxc-ls -1 | grep utility | head -n 1) "
+                     "-- bash -c '. /root/openrc ; ")
+
+random_str = helpers.generate_random_string(6)
+instance_name = "test_instance_{}".format(random_str)
+image_name = 'Cirros-0.3.5'
+getway_net = 'GATEWAY_NET'
+private_net = 'PRIVATE_NET'
+flavor = 'm1.tiny'
+floating_ip_name = None
+
+
+@pytest.mark.test_id('bea6da62-7968-11e8-a634-9cdc71d6c128')
```

Comment:
This test duplicates id ''26aa7902-53da-11e8-96e0-6a0003552100" from https://github.com/rcbops/molecule-validate-nova-deploy/blob/master/molecule/default/tests/test_create_snapshot_on_instance.py#L10-L13

Is there a way to pull in the test without creating an exact duplicate with a different uuid?